src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -420,15 +420,15 @@ public FeedbackSessionQuestionsBundle getFeedbackSessionQuestionsForStudent(
         List<FeedbackQuestionAttributes> questions = fqLogic.getFeedbackQuestionsForStudents(feedbackSessionName,
                 courseId);
 
-
+        Set<String> hiddenInstructorEmails = getHiddenInstrutorEmails(courseId);
```

Comment:
The overhead of performing the extra database queries is likely larger than looping through the questions twice (and it costs money). But sure, you don't have to do it if you don't want to, the query for the instructors probably isn't that expensive.